### PR TITLE
[plugin.onedrive] 1.1.8

### PR DESCRIPTION
**Pull request by cguZZman**

### CODE REVIEW

File: plugin.onedrive/addon.py [hunk 1]
```diff
@@ -411,14 +446,60 @@ def report_error(e):
...
+                    remote_user_id = utils.Utils.unicode(remote_user['id'])
+                    if remote_user_id not in user_dic:
+                        user_dic[remote_user_id] = [f]
+                        list_item = xbmcgui.ListItem(utils.Utils.unicode(remote_user['displayName']))
+                        params = {'action':'open_shared_by', 'content_type': content_type, 'remote_user_id': remote_user_id, 'driveid': driveid}
+                        url = base_url + '?' + urllib.urlencode(params)
+                        xbmcplugin.addDirectoryItem(addon_handle, url, list_item, True)
+                    else:
+                        user_dic[remote_user_id].append(f)
+            fo = open(shared_json_path, 'wb')
```
Comment (phil65) [inline plugin.onedrive/addon.py]: please use "with" statement for opening files.

File: plugin.onedrive/addon.py [hunk 2]
```diff
@@ -411,14 +446,60 @@ def report_error(e):
...
+            fo.write(json.dumps(user_dic))
+            fo.close()
+        if not cancelOperation(onedrive):
+            xbmcplugin.endOfDirectory(addon_handle)
+    elif action[0] == 'open_shared_by':
+        driveid = args.get('driveid')[0]
+        onedrive = onedrives[driveid]
+        remote_user_id = args.get('remote_user_id')[0]
+        fo = open(shared_json_path, 'rb')
```
Comment (phil65) [inline plugin.onedrive/addon.py]: same here.

File: plugin.onedrive/resources/language/english/strings.xml [hunk 3]
```diff
@@ -52,4 +52,11 @@
     <string id="30049">Loading %s items in this folder, please wait...</string>
     <string id="30050">Loading %s items from search, please wait...</string>
     <string id="30051">When playing videos, set the subtitle file located next to the video (.srt)</string>
+    <string id="30052">Files</string>
```
Comment (phil65) [inline plugin.onedrive/resources/language/english/strings.xml]: Add-on string id range is from 32000-33000. Please adjust it when you find the time, and also convert the xmls to .po files.

**Comment by cguZZman** [comment]
Change to use "with" statement: Done.
I'll adjust the string file and convert it to .po files in the next release.
Thanks!

**Pull request merged**